packages/calling-stateful-client/src/RaiseHandSubscriber.ts

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(raise-hand) */
+import { RaiseHandCallFeature, RaisedHandChangedEvent } from '@azure/communication-calling';
+/* @conditional-compile-remove(raise-hand) */
+import { CallContext } from './CallContext';
+/* @conditional-compile-remove(raise-hand) */
+import { CallIdRef } from './CallIdRef';
+/* @conditional-compile-remove(raise-hand) */
+import { toFlatCommunicationIdentifier } from '@internal/acs-ui-common';
+/* @conditional-compile-remove(raise-hand) */
+/**
+ * @private
+ */
+export class RaiseHandSubscriber {
+  private _callIdRef: CallIdRef;
+  private _context: CallContext;
+  private _raiseHand: RaiseHandCallFeature;
+
+  constructor(callIdRef: CallIdRef, context: CallContext, raiseHand: RaiseHandCallFeature) {
+    this._callIdRef = callIdRef;
+    this._context = context;
+    this._raiseHand = raiseHand;
+
+    this.subscribe();
```

Comment:
if all of the constructors call subscribe then this should be fine. The problem exists where people assume the constructor is not subscribing which can cause unintended issues.